cancermuts/datasources.py

```diff
@@ -43,6 +43,7 @@
 import sys
 import os
 import myvariant
```

Comment:
we don't need this import anymore

please also check that

  - requirements in `setup.py` are still consistent with the code as it is now (which I don't think they are right now). We need the package to still install properly
  - if we need any change in the documentation following this